testing/deploy_kubeflow.py

```diff
@@ -0,0 +1,127 @@
+"""Deploy Kubeflow and wait for it to be deployed.
+
+TODO(jlewi): This script is outdated. Its no longer used for GKE.
+It is still used by minikube. For minikube we should be using kfctl.sh
```

Comment:
Did you mean "for GKE we should be using kfctl.sh"?

Reply:
No I meant we should switch to using kfctl.sh for our minikube E2E test; at which point we shouldn't need this script anymore. GKE already uses kfctl.sh to deploy so its not using this script.